java-rest-service/accelerator.yaml

```diff
@@ -57,22 +52,22 @@ accelerator:
     - name: apiSystem
       label: System API Belongs To
       inputType: text
-      required: false
-      defaultValue: ''
+      required: true
+      defaultValue: "Profile Management"
       dependsOn:
         name: exposeOpenAPIEndpoint
     - name: apiOwner
       label: Owner of API
       inputType: text
-      required: false
-      defaultValue: ''
+      required: true
+      defaultValue: "Customer Relations Department"
```

Comment:
owners would be user or group I assume and they are typically lower case or kebab-case

Reply:
Done